fastbril/benchmarks/example.toml

```diff
@@ -0,0 +1,21 @@
+extract = 'rtime: (\d+)'
+benchmarks = '*.bril'
+epsilon = 0.000000001
+
+[runs.baseline]
+pipeline = [
+    "bril2json",
+    "\\time -f \"rtime: %R\" brili -p {args}",
```

Comment:
It `time -f` standard? I don't seem to have this on mac.

Reply:
-f is standard afaik, but zsh has its own time built-in which is probably what you are seeing on Mac

Reply:
I'm a hold out that still uses bash on Mac, but fair. Didn't have too much luck with `/usr/bin/time` but its probably just an issue on my end

Reply:
I will say all of this is only tested on Linux and the benchmark stuff was mostly for the report, so it might make sense to just remove it here

Reply:
i told make to use `/bin/sh` maybe this will fix it?